client/components/utils/placeholder.js

```diff
@@ -1,44 +1,48 @@
 import {Plugin} from 'prosemirror-state';
 import {Decoration, DecorationSet} from 'prosemirror-view';
 
-export default new Plugin({
-  props: {
-    decorations: (state) => {
-      const decorations = [];
-      let isThereParagraphWithContent = false;
-      let emptyParagraph = null;
+export const PlaceholderPlugin = (vueInstance, titlePlaceholder, paragraphPlaceholder) => {
```

Comment:
Code style. `PlaceholderPlugin` is not a function name. See `menu.js` for examples.